### DISCUSSION TITLE
Email for education student developer pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#147066] [GitHub Education]
### Hi everyone,

I have already registered and verified my institution mail id with my github account but when i try to register for the studentt developer pack it says that please add and verify your institution mail id, i have already tried removing and adding again but it still does't work what i can do
![image_2024-12-14_113938837](https://github.com/user-attachments/assets/87d65b04-1259-4255-98e9-61fd60ca9685)
![image_2024-12-14_114024816](https://github.com/user-attachments/assets/a53ddd06-c44d-4720-8f71-a04738a79919)

**Comment by peter900932** [comment]
I’ve registered and verified my institutional email with my GitHub account, but when I try to apply for the Student Developer Pack, it asks me to add and verify my email again. I’ve already tried removing and re-adding the email, but it still doesn’t work. What can I do to resolve this issue?

**Reply by ghost** [reply]
yeah i too have the same problem
